sys-apps/msr-tools/msr-tools-1.3_p20170320.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CONFIG_CHECK="~X86_CPUID ~X86_MSR"
+inherit autotools linux-info
+
+REV="eec71d977a83f8dc76bc3ccc6de5cbd3be378572"
```

Comment:
I would suggest using EGIT_COMMIT instead of REV, since most packages to use that.